cli/lib/kontena/cli/services/services_helper.rb

```diff
@@ -287,41 +287,45 @@ def deploy_service(token, service_id, data)
 
         # @param [Hash] deployment
         # @return [String] multi-line
-        def render_service_deploy_instances(deployment)
+        def render_service_deploy_instances(deployment, vocabulary)
           deployment['instance_deploys'].map{ |instance_deploy|
-            description = "instance #{deployment['service_id']}-#{instance_deploy['instance_number']} to node #{instance_deploy['node']}"
+            description = "instance #{deployment['service_id']}-#{instance_deploy['instance_number']} #{vocabulary[:preposition]} node #{instance_deploy['node']}"
 
             case instance_deploy['state']
             when 'created'
-              "#{pastel.dark('⊝')} Deploy #{description}"
+              "#{pastel.dark('⊝')} #{vocabulary[:verb]} #{description}"
             when 'ongoing'
-              "#{pastel.cyan('⊙')} Deploying #{description}..."
+              "#{pastel.cyan('⊙')} #{vocabulary[:verb_ing]} #{description}..."
             when 'success'
-              "#{pastel.green('⊛')} Deployed #{description}"
+              "#{pastel.green('⊛')} #{vocabulary[:verb_ed]} #{description}"
             when 'error'
-              "#{pastel.red('⊗')} Failed to deploy #{description}: #{instance_deploy['error']}"
+              "#{pastel.red('⊗')} Failed to #{vocabulary[:verb]} #{description}: #{instance_deploy['error']}"
             else
-              "#{pastel.dark('⊗')} Deploy #{description}?"
+              "#{pastel.dark('⊗')} #{vocabulary[:verb]} #{description}?"
             end
           }
         end
 
-        # @param [String] token
         # @param [Hash] deployment
         # @param [Fixnum] timeout
         # @param [Boo lean] verbose
         # @raise [Kontena::Errors::StandardError]
-        def wait_for_deploy_to_finish(token, deployment, timeout: 600)
+        def wait_for_deploy_to_finish(deployment, timeout: 600, vocabulary: {
+              :verb => "Deploy",
```

Comment:
Multi-line `def` = yuck.

Reply:
The casing here was also breaking the e2e specs... titlecase verb

```
  3) service deploy For a service that fails to deploy fails to deploy with an error
     Failure/Error: expect(k.out).to match /Failed to deploy instance .+ to node .+: GridServiceInstanceDeployer::ServiceError: .*stat \/dev\/null\/wtf: not a directory/
       expected "    *   Deploying service \e[36mtest-fail\e[0m  .. \\\r                                         \r\r...y onto a file (or vice-versa)? Check if the specified host path exists and is the expected type\r\n" to match /Failed to deploy instance .+ to node .+: GridServiceInstanceDeployer::ServiceError: .*stat \/dev\/null\/wtf: not a directory/
       Diff:
       @@ -1,2 +1,5 @@
       -/Failed to deploy instance .+ to node .+: GridServiceInstanceDeployer::ServiceError: .*stat \/dev\/null\/wtf: not a directory/
 [fail] Deploying service test-fail      
       + [error] 500 : halting deploy of e2e/null/test-fail, one or more instances failed:
       +	⊗ Failed to Deploy instance e2e/null/test-fail-1 to node travis-job-5ef35947-5850-45ee-9643-cb3f3cbf289f: GridServiceInstanceDeployer::ServiceError: Docker::Error::ClientError: oci runtime error: container_linux.go:265: starting container process caused "process_linux.go:368: container init caused \"rootfs_linux.go:57: mounting \\\"/dev/null/wtf\\\" to rootfs \\\"/var/lib/docker/overlay2/689b25a9ee05b6bfc78f48e0aab2597d653c6c8578421d55136020c4657d0214/merged\\\" at \\\"/dev/wtf\\\" caused \\\"stat /dev/null/wtf: not a directory\\\"\""
       +: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
     # ./spec/features/service/deploy_spec.rb:25:in `block (3 levels) in <top (required)>'
```